prog/dftb+/lib_timedep/dynamics.F90

```diff
@@ -392,6 +428,12 @@ subroutine TElecDynamics_init(this, inp, species, speciesName, tWriteAutotest, a
     this%kPoint = kPoint
     this%KWeight = KWeight
     allocate(this%parallelKS, source=parallelKS)
+    allocate(this%populDat(this%parallelKS%nLocalKS))
+    if (.not. allocated(sccCalc)) then
```

Comment:
Should make the indent clearer, but might be a good candidate for an assertion rather than a runtime error
```suggestion
    if (.not. present(sccCalc)) then
```

Reply:
Same as above, superseded by https://github.com/dftbplus/dftbplus/pull/657